starter_code/views/random-beers.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="randomBeer">
+    <img src="{{image_url}}" alt="Beer bottle image" class="productImage">
+    <h1 class="beerName">{{name}}</h1>
+    <h2 class="tagline">{{tagline}}</h2>
+    <p class="description">{{description}}</p>
+    <h2>Food Pairing</h2>
+    <p class="food_pairing">{{food_pairing}}</p>
```

Comment:
for the food pairing to appear like in the example, you can loop over the food pairing array 
```
<ul>
{{#each food_pairing}} 
   {{this}}
<li>{{/each }}</li>
</ul> 
```